### ISSUE TITLE
Remove "save as variable" suggestion every friggin time I select a piece of text!

### ISSUE THREAD

**Issue by Neboyarsky** [a85/POSTMan-Chrome-Extension#12080]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

I need Postman app to stp showing the popup "set as variable" every damn time I select a piece of text in request URL, parameters or literally any other input!
![image](https://github.com/postmanlabs/postman-app-support/assets/44096384/192f2b3b-bc70-4588-b4e7-7d0cf4d61bfb)

I only need to select the text to copy it or to delete a piece or replace with something else, why do you overlap half of the selection with a popup that I will never use?

### Describe the solution you'd like

Just stop this crap from happening or make a configurable option to enable it, and keep it OFF by default!

**Comment by Neboyarsky** [comment]
Make it a context menu action, for G-d's sake!

**Comment by giridharvc7** [comment]
@Neboyarsky i understand it can be a bit frustrating. Thank you for the feedback, we'll look into how we can better this experience.

**Comment by ZeroBladeTV** [comment]
Came here to say - PLEASE! This has also been driving me nuts.

**Comment by anshunjain** [comment]
this is god damn frustrating please change this! +1

**Comment by evilduck95** [comment]
+1 this is especially annoying when I make the Params or Headers boxes very small and try to select text to copy!

**Comment by Neboyarsky** [comment]
@giridharvc7 has there been any traction towards removing this annoying behaviour or making it optional?

**Comment by devwax** [comment]
+1 - Maybe add a toggle setting in **Settings > General > User Interface**. e.g. "Offer to set selected text as a variable on select."
![image](https://github.com/postmanlabs/postman-app-support/assets/1509662/00f030f7-db2d-4e04-8e74-6945051e48da)

**Comment by devwax** [comment]
Also, keep the field focused when alt/cmd-tabbing away from Postman. Now you have to re-select any field that you were typing in when you switch back to the app.

**Comment by fwend** [comment]
I agree, it would be nice if we could turn it off. Sometimes I click it by accident, it can be a bit distracting. Thanks!

**Comment by kondratieph** [comment]
For the God's sake, just remove it!

**Comment by lukas-pierce** [comment]
It's been more than a year since this request was created, please remove this annoying popup

**Comment by St00pKid** [comment]
adding onto this request. I'm in my global variable screen trying to edit a long secret already saved as a variable and the popup is blocking my view of the text I'm trying to edit when the text box is expanded.

I understand you're trying to advertise a feature some people don't know about but please give the option to disable.

**Comment by skatikia** [comment]
A little late to be closing this issue. But the 'Set as variable' option should no longer be appearing for you when you select a text in Postman. We kind of found it annoying, too : (

Instead, you should be able to access this option by right-clicking on a selection. Thank you so much y'all for your feedback.

<img width="674" height="437" alt="Image" src="https://github.com/user-attachments/assets/36950247-4ee6-4514-ab88-0d6598be0368" />
